README.md

```diff
@@ -1,4 +1,4 @@
-# AspirePress Updater Plugin
+# AspirePress Update Plugin
```

Comment:
The  AspirePress update in  was removed in https://github.com/aspirepress/AspireUpdate/commit/71f997c23f72e03671d07b9c0db219ad685877d4

Is it correct this should change again?